apps/users/models.py

```diff
@@ -9,8 +9,14 @@
 from rest_framework.authtoken.models import Token
 
 
+class Role(models.TextChoices):
```

Comment:
```suggestion
class UserRole(models.TextChoices):
```
이런식으로 조금더 명시적이면 좋을 것 같습니다

Reply:
넵